tests/testthat/test-lassnags.R

```diff
@@ -0,0 +1,17 @@
+context("lassnags")
```

Comment:
Idem that's a too long computation. CRAN will complain.

Reply:
Should I just delete it then?

Reply:
The dataset you send me by email is nice. Compressed in laz its weight dropped to 500 kB and the tree segmentation is easy and therefore the data is suitable in replacement of the current file used as example `Tree.laz`.

However it is still too big for two reasons:
* The total weight of example files is greater than 5 MB
* The time spent of the `lastrees` example (using li 2012) is too long and the time spent on `lassnag` is too long too.

The solution is to decimate your dataset to 5 pt/m² and crop it a bit. Thus the dataset is still good for an example of tree segmentation (good sementation in less than 1 second) and snags detection (output is almost the same after decimation). 

But `lassnags` is still too slow to be tested by the CRAN. So the example must be wrapped in `\dontrun`. And the test in `testthat` must be removed (commented only, thus if we speed up the algo we can simply uncomment).

And you must review your example. It does not work. Also run `devtools::check()` before a PR to check everything. You should have 0 error, 0 warning, 0 notes (actually the devel branch have currently some warnings. Check only those which concern `lassnags`)

Reply:
Gotcha... I'm on it asap